metricbeat/module/php_fpm/process/_meta/fields.yml

```diff
@@ -29,7 +29,6 @@
     type: integer
     description: >
       The duration in microseconds (1 million in a second) of the current request (my own definition)
-  # ECS?
   - name: request_method
```

Comment:
the php-fpm module needs another good look. Many things can map to ECS in there.

pid, request_uri => url.original, content_length => http.request.body.size, request_duration => event.duration

Reply:
Ok disregard this comment. Hadn't seen https://github.com/elastic/beats/pull/10366 yet :-) Resolving